### ISSUE TITLE
Latest release : SIGSEGV

### ISSUE THREAD

**Issue by bluskye** [linuxserver/docker-qbittorrent#148]
Hi
Qbittorent web UI is not accessible.  Logs below, I assume this is release-4.4.0-ls167.  I am using latest tag and redeployed it today Jan 10, noon America/Chicago time.

## Docker logs from Portainer

******** Information ********
To control qBittorrent, access the WebUI at: http://localhost:8080
*************************************************************
Please file a bug report at http://bug.qbittorrent.org and provide the following information:
qBittorrent version: v4.4.0
Caught signal: SIGSEGV

Stack trace:

**Comment by j0nnymoe** [comment]
Please provide the information requested within the template.

**Comment by bluskye** [comment]
## Expected Behavior
Web UI should appear on port 8080

## Current Behavior
Web UI is not loading

## Steps to Reproduce
Deploy qbittorrrent with the following stack:

version: "2.1"
services:
  qbittorrent:
    image: lscr.io/linuxserver/qbittorrent:latest
    container_name: qbittorrent
    environment:
      - PUID=1000
      - PGID=100
      - TZ=America/Chicago
      - WEBUI_PORT=8080
    volumes:
      - /srv/dev-disk-by-uuid-2df41948-e5c4-44b4-ad1d-fda43b2bd0c1/docker/qbittorrent:/config
      - /srv/dev-disk-by-uuid-b3e06e03-6d90-406e-a4a8-62a62d894e26/downloads/qbittrrent:/downloads
    ports:
      - 6881:6881
      - 6881:6881/udp
      - 8080:8080
    restart: unless-stopped

## Environment
**OS: Debian Buster (Open Mediavault)
**CPU architecture:** x86_64
**How docker service was installed:**
Portainer through stacks, see above

## Command used to create docker container (run/create/compose/screenshot)

version: "2.1"
services:
  qbittorrent:
    image: lscr.io/linuxserver/qbittorrent:latest
    container_name: qbittorrent
    environment:
      - PUID=1000
      - PGID=100
      - TZ=America/Chicago
      - WEBUI_PORT=8080
    volumes:
      - /srv/dev-disk-by-uuid-2df41948-e5c4-44b4-ad1d-fda43b2bd0c1/docker/qbittorrent:/config
      - /srv/dev-disk-by-uuid-b3e06e03-6d90-406e-a4a8-62a62d894e26/downloads/qbittrrent:/downloads
    ports:
      - 6881:6881
      - 6881:6881/udp
      - 8080:8080
    restart: unless-stopped

## Docker logs


*************************************************************
Please file a bug report at http://bug.qbittorrent.org and provide the following information:

qBittorrent version: v4.4.0

Caught signal: SIGSEGV

Stack trace:

******** Information ********
To control qBittorrent, access the WebUI at: http://localhost:8080



*************************************************************
Please file a bug report at http://bug.qbittorrent.org and provide the following information:

qBittorrent version: v4.4.0

Caught signal: SIGSEGV

Stack trace:

******** Information ********
To control qBittorrent, access the WebUI at: http://localhost:8080



*************************************************************
Please file a bug report at http://bug.qbittorrent.org and provide the following information:

qBittorrent version: v4.4.0

Caught signal: SIGSEGV

Stack trace:

******** Information ********
To control qBittorrent, access the WebUI at: http://localhost:8080



*************************************************************
Please file a bug report at http://bug.qbittorrent.org and provide the following information:

qBittorrent version: v4.4.0

Caught signal: SIGSEGV

Stack trace:

******** Information ********
To control qBittorrent, access the WebUI at: http://localhost:8080



*************************************************************
Please file a bug report at http://bug.qbittorrent.org and provide the following information:

qBittorrent version: v4.4.0

Caught signal: SIGSEGV

Stack trace:

******** Information ********
To control qBittorrent, access the WebUI at: http://localhost:8080



*************************************************************
Please file a bug report at http://bug.qbittorrent.org and provide the following information:

qBittorrent version: v4.4.0

Caught signal: SIGSEGV

Stack trace:

******** Information ********
To control qBittorrent, access the WebUI at: http://localhost:8080



*************************************************************
Please file a bug report at http://bug.qbittorrent.org and provide the following information:

qBittorrent version: v4.4.0

Caught signal: SIGSEGV

Stack trace:

******** Information ********
To control qBittorrent, access the WebUI at: http://localhost:8080



*************************************************************
Please file a bug report at http://bug.qbittorrent.org and provide the following information:

qBittorrent version: v4.4.0

Caught signal: SIGSEGV

Stack trace:

**Comment by bluskye** [comment]
Went back to tag amd64-14.3.9 and its working again.

**Comment by alexbelgium** [comment]
There are many issues with the latest qbittorrent release, has shown on their own github issue tracker (one of those being a change in the SavePath tag, that causes many issues). Imo, it is preferable to avoid the 4.4.0 release until they release the next app update

**Comment by bluskye** [comment]
> There are many issues with the latest qbittorrent release, has shown on their own github issue tracker (one of those being a change in the SavePath tag, that causes many issues). Imo, it is preferable to avoid the 4.4.0 release until they release the next app update

Been trying Qbitorrent for a week or so, there are major issues with this app in docker.  I went back to latest, it booted but I consistently see crazy run-away swap usage.  Its crazy how much swap qbittorrent uses idling after a torrent is complete.  Transmission barely uses anything sitting idle and seeding.  See below.  Going back to transmission.


Intel(R) Xeon(R) CPU E3-1265L v3 @ 2.50GHz
CPU	        3.8%
MEM	25.3%
SWAP	75.7%

Name	Status	CPU%	MEM	RSS	IOR/s	IOW/s	RX/s	TX/s	Command
glances	running	1.4	356M	256M	0b	0b	19Kb	82Kb	["/bin/sh","-c","python3 -m glances -C /glances/conf/glances.conf $GLANCES_OPT"]
qbittorrent	running	9.2	5.43G	28.0M	7.37Mb	0b	76Kb	5.01Mb	["/init"]
jellyfin	running	4.2	216M	167M	0b	0b	0b	4Kb	["/init"]
sonarr	running	0.0	143M	131M	0b	0b	0b	0b	["/init"]
homebridge	running	0.5	128M	102M	0b	0b	NaNb	NaNb	["/init"]
portainer	running	0.0	27.9M	17.3M	0b	0b	0b	0b	["/portainer"]
watchtower	running	0.0	18.0M	14.5M	0b	0b	0b	0b	["/watchtower"]
transmission	running	0.2	40.9M	4.30M	0b	0b	360b	2Kb	["/init"]

**Comment by drizuid** [comment]
tl;dr report app issues to the app devs as the log suggests.

much as the errors in the logs show, the bugs should be reported upstream. The issues are with the app, not the container. In our opinion, barring known bugs in the application itself, it works fine
![image](https://user-images.githubusercontent.com/40674481/150154355-385746a7-d3dd-493f-86bf-be8b3fe41eeb.png)
